apl/tabular-operators/project-reorder-operator.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: 'project-reorder'
+description: 'This page explains how to reorder specified fields in the output.'
+tags: 
+    ['axiom documentation', 'documentation', 'axiom', 'tabular operators', 'project-reorder']
+---
+
+Reorders specified fields in the output while keeping the original order of unspecified fields.
+
+## Syntax
+
+```kusto
+| project-reorder FieldName1, FieldName2, ... [asc|desc]
```

Comment:
This should be: `project-reorder FieldName1OrWildcard [asc|desc], FieldName2OrWildcard [direction], ...`

Reply:
we support wildcard suffix matches so probably more 

> `project-reorder FieldName[*] [asc|desc|granny-asc|granny-desc], ...`

both of these are valid 
```kql
T
| project-reorder testing* desc // orders all the fields starting with testing first, then all the other fields
``` 

```kql
T
| project-reorder trace_id, *, span_id // orders the trace_id then everything else, then span_id fields 
```